.ci/build-app-image.sh

```diff
@@ -26,6 +30,8 @@ ENV DNX_TRACE 1
 ENTRYPOINT sleep 10000 | dnx . kestrel
```

Comment:
Since all the samples(including older beta versions with their own respective `dnx . kestrel`) in the home repo have their own Dockerfile, I think the we can modify this line to be `dnx kestrel` instead and going forward this is going to be the way to run.